blog/2018-01-09-smart-testing-for-fast-feedback.textile

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Smart Testing For Fast Feedback
+author: hemanik
+tags: [ testing, guide, smart-testing ]
+---
+
+h1. Smart Testing For Fast Feedback
```

Comment:
shouldn't that be `Smart Testing For Fast Feedback Loop` or maybe just `Smart Testing`?